src/screens/Forum/ForumSubcategoryPostsScreen.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
```

Comment:
This file is similar to ForumHomeScreen right? is there any way we can factor out the common code so we can share it?